src/app/organizations/organizations/organizations.component.html

```diff
@@ -20,59 +20,90 @@
   </div>
 </app-header>
 <div *ngIf="!(loading$ | async)" class="container">
-  <button
-    *ngIf="isLoggedIn$ | async"
-    class="pull-right"
-    color="primary"
-    mat-raised-button
-    (click)="createOrganization()"
-    matTooltip="Create Organization Request"
-  >
-    Create Organization Request
-  </button>
-  <div *ngIf="filteredOrganizations$ | async as organizations; else noOrganizationsFound">
-    <form [formGroup]="organizationSearchForm">
-      <mat-form-field>
-        <input matInput placeholder="Search organizations" formControlName="name" />
-      </mat-form-field>
-    </form>
+  <div *ngIf="obs | async as organizations; else noOrganizationsFound">
```

Comment:
`obs` is the observable I messed around with paginator in the `.ts` file, right now it doesn't work. `obs` is dead